### DISCUSSION TITLE
Typograhy: How to change the color of the line in the blockquote

### DISCUSSION THREAD

**Discussion by srotread** [tailwindlabs/tailwindcss#11446] [Help]
The line on the left of the blockquote, has a light accent color by default. I have no idea where this is coming from and I would like to change it. Is there a utility for the selector of this or maybe it needs to changed in the config.

<img width="758" alt="image" src="https://github.com/tailwindlabs/tailwindcss/assets/136879970/a1723771-f31e-40be-b13c-8d2fe166b710">

**Comment by wongjn** [comment]
The light accent color is a left border as applied by the [CSS from the typography plugin](https://github.com/tailwindlabs/tailwindcss-typography/blob/4c3b76ccc8120f307514d5721598763fd09761c0/src/styles.js#L1311-L1312):

```js
module.exports = {
  DEFAULT: {
    css: [
      {
        // …
        blockquote: {
          // …
          borderLeftWidth: '0.25rem',
          borderLeftColor: 'var(--tw-prose-quote-borders)',
          // …
        },
        // …
      },
      // …
    ],
  },
};
```
You can change it in several ways:

- Use a utility class on the `prose` element targeting `blockquote` elements:
  ```html
  <div class="prose prose-blockquote:border-l-red-500">
  ```
  Or for only the border color, you could change the CSS variable:
  ```html
  <div class="prose [--tw-prose-quote-borders:theme(colors.red.500)]">
  ```
- [Customize the CSS in configuration](https://tailwindcss.com/docs/typography-plugin#customizing-the-css):
  ```js  /** @type {import('tailwindcss').Config} */
  module.exports = {
    theme: {
      extend: {
        typography: ({ theme }) => ({
          DEFAULT: {
            css: {
              blockquote: {
                borderLeftColor: theme('colors.red.500'),
              }
              // Or
              '--tw-prose-quote-borders': theme('colors.red.500'),
            },
          },
        }),
      },
    },
    plugins: [
      require('@tailwindcss/typography')
    ],
  }
  ```
